data/mods/Magiclysm/worldgen/orc_village.json

```
@@ -0,0 +1,65 @@
+[
+  {
+    "type": "palette",
+    "id": "orc_palette",
+    "terrain": { "3": "t_door_makeshift_c", "1": "t_wall_rammed_earth", ".": "t_dirtfloor" }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "orc_village",
+    "object": {
+      "fill_ter": "t_dirt",
+      "rows": [
+        "111111111111111111111111",
+        "1......................1",
+        "1......................3",
+        "1......................1",
+        "1......................3",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................3",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "1......................1",
+        "111111111111111111111111"
+      ],
+      "palettes": [ "orc_palette" ],
+      "place_monsters": [
+        { "monster": "GROUP_ORC", "x": [ 3, 5 ], "y": [ 4, 23 ], "density": 0.2, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_ORC", "x": [ 3, 21 ], "y": [ 4, 23 ], "density": 0.2, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_ORC", "x": [ 3, 21 ], "y": [ 4, 23 ], "density": 0.2, "repeat": [ 1, 3 ] },
+        { "monster": "GROUP_ORC", "x": [ 3, 21 ], "y": [ 4, 23 ], "density": 0.2, "repeat": [ 1, 3 ] }
+      ]
+    }
+  },
+  {
+    "type": "overmap_special",
+    "id": "orc_village",
+    "overmaps": [ { "point": [ 0, 0, 0 ], "overmap": "orc_village_north" } ],
+    "locations": [ "wilderness" ],
+    "city_distance": [ 20, -5 ],
+    "city_sizes": [ 0, 13 ],
+    "occurrences": [ 0, 6 ]
+  },
+  {
+    "type": "overmap_terrain",
+    "id": [ "orc_village" ],
+    "name": "orc_village",
+    "sym": "o",
+    "color": "green",
+    "see_cost": 6
```

```suggestion
    "see_cost": 6,
    "flags": [ "RISK_HIGH" ]
```
This means that NPCs will consider it a higher-risk area (to me, this makes sense given that the orc village locations are visible on the overmap, implying that others will know where they are).